### DISCUSSION TITLE
failed to fetch dynamically imported module after loading other successfully in vite project

### DISCUSSION THREAD

**Discussion by tarikrital** [quasarframework/quasar#13790] [General - Components / Directives / etc] [answered]
I have a Quasar Vite project using TS and routes defined the same way using `layouts` `pages` dirs.. But for one component it gives the following error!

![image](https://user-images.githubusercontent.com/52907282/175810246-6a90130f-f801-4b85-901e-8bdc2de071dc.png)

**Comment by MilosPaunovic** [comment]
Check the component import statement.

**Reply by tarikrital** [reply]
How that @MilosPaunovic! can you mention a tip?

> and routes defined the same way using layouts pages dirs

FYI other components work.

![image](https://user-images.githubusercontent.com/52907282/175825742-74e1355e-a8d0-4ff9-b5f3-32b3b99972e3.png)

**Reply by metalsadman** [reply]
restart your dev server.

**Reply by tarikrital** [reply]
I did but the same result!

**Reply by MilosPaunovic** [reply]
Seems like there is an issue with the path of the component you're providing to the router import. Try to check that, and if it does not resolve the problem then you're gonna need to provide reproduction repository for someone to take a deeper dive into it.

**Reply by tarikrital** [reply]
It was working with `Webpack` but after migration to `Vite` it raises that issue, maybe a problem with router-link?

**Reply by MilosPaunovic** [reply]
Without a proper code reproduction there isn't much more help possible. There shouldn't be a problem related to Vite which efects only one component. Have you checked import path?

If you run out of options, provide a reproduction repository so someone could have a deeper dive.

**Comment by tarikrital** [comment, marked as answer]
Seems like it was a **browser cache** issue but **unsure** because after I changed the port to `8082` worked and also it was an issue with changing `import ArtVideoPlayer from 'components/ArtVideoPlayer'`
to `import ArtVideoPlayer from 'components/ArtVideoPlayer.vue'` in other _SFC_ not listed in `routes.ts` which was related to .vue extension.

**Reply by metalsadman** [reply]
yes in vite you must include the `.vue` extension.
